node/src/GlideClient.ts

```diff
@@ -371,6 +372,37 @@ export class GlideClient extends BaseClient {
         return this.createWritePromise(createTime());
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace`
+     * is true, removes the `destination` key first if it already exists, otherwise performs no action.
```

Comment:
Same here, can we copy what Python has?
```suggestion
     * Copies the value stored at the `source` to the `destination` key. If `destinationDB`
     * is specified, the value will be copied to the database specified by `destinationDB`,
     * otherwise the current database will be used. When `replace`
     * is true, removes the `destination` key first if it already exists, otherwise performs no action.
```